tests/integrations/mcs/members/member_test.go

```diff
@@ -65,7 +66,7 @@ func (suite *memberTestSuite) SetupTest() {
 
 	// TSO
 	nodes := make(map[string]bs.Server)
-	for i := 0; i < 3; i++ {
+	for i := 0; i < constant.DefaultKeyspaceGroupReplicaCount; i++ {
```

Comment:
why reduce it?

Reply:
Because `DefaultKeyspaceGroupReplicaCount` limits the number of `tso` nodes.

Reply:
We do need to test the case whose TSO node is more than `DefaultKeyspaceGroupReplicaCount`.

Reply:
Added restrictions and checks to the `TransferPrimary` function and refined the tests.